docs/source/markdown/podman.1.md

```diff
@@ -114,6 +114,8 @@ Overriding this option will cause the *storage-opt* settings in `containers-stor
 Storage state directory where all state information is stored (default: "/run/containers/storage" for UID 0, "/run/user/$UID/run" for other users).
 Default state dir configured in `containers-storage.conf(5)`.
 
+Note: Following flag is non-functional if backend database is already initialized.
```

Comment:
I think we refer to options in the docs and not use the term flag. Also I don't think it clear for users what backend database means. It should be clear that option ha only effect on the first podman command or when --root is set to a new path. If the db is found under --root we will always use the paths from there.

needs to add the same note to --tmpdir and --storage-driver